astro/src/content/docs/customize/look-and-feel/upgrade.mdx

```diff
@@ -12,6 +12,366 @@ New versions of FusionAuth sometimes include new or updated theme templates. If
 
 This page contains notes on changes in recent versions of FusionAuth that require changes to a customized theme.
 
+## Version 1.62.0
```

Comment:
It's going to be really hard to understand what's changing here. Is there a way to summarize the changes?